localization/src/languages/de.json

```diff
@@ -89,7 +90,8 @@
             "PickCipher": "Pick from encryption keys",
             "CipherKeys": "Encryption keys",
             "AddCipherKey": "Add an encryption key",
-            "CipherKeyLabel": "Encryption key"
+            "CipherKeyLabel": "Encryption key",
+            "Encrypt": "Encrypt"
```

Comment:
It's not necessary `encrypt`, as the could also be used for decryption....

Reply:
https://github.com/tonlabs/UIKit/pull/1476/commits/9a0447c5cc35417b0ecb3622a1aaebf571c8e06b